fix issues

LastVegass opened the pull request:
- fix style

LastVegass committed:
fix issues

- fix style

goblynwtf reviewed (changes requested):
Optimize to_i usage

LastVegass committed:
fix issues

- fix style & methods

goblynwtf commented on ruby.rb:
```diff
@@ -38,32 +38,32 @@ def rating_to_stars(rating)
-  movies.map { |m| m[:name].length }
-        .sort.reverse
-        .select { |m| m[:name].length > rightsize[5] }
+  size = movies.map { |m| m[:name].length }
```
можно объединить в чейн

goblynwtf commented on ruby.rb:
```diff
@@ -38,32 +38,32 @@ def rating_to_stars(rating)
-        .sort
-        .select { |m| m[:realise].to_i < rightdate[10] && m[:geners].include?('Comedy') }
+def movies_comedy(movies)
+  comedylist = movies.select { |m| m[:geners].include?('Comedy') }
```
снова много переменных, в чейн это дело

goblynwtf commented on ruby.rb:
```diff
@@ -38,32 +38,32 @@ def rating_to_stars(rating)
+  comedylist = movies.select { |m| m[:geners].include?('Comedy') }
+  date = comedylist.map { |m| m[:realise] }
+  rightdate = date.sort
+  movies.select { |m| m[:realise] < rightdate[10] && m[:geners].include?('Comedy') }
```
почему `rightdate` 10?

goblynwtf commented on ruby.rb:
```diff
@@ -38,32 +38,32 @@ def rating_to_stars(rating)
   movies.select do |m|
     unless m[:country].include?('USA')
-      puts m[:name], m[:realise], m[:geners], m[:runtime], m[:stars].chomp!
+      puts "#{m[:name]} (#{m[:realise]} ; #{m[:geners]}) - #{m[:runtime]} "
```
`puts` тут не нужен, уже обсуждали это :)

LastVegass committed:
fix issues

fix 2 methods(comedy & length)

LastVegass committed:
fix issues

- edit movies_not_usa function

LastVegass committed:
fix issues

- edit director_list function
- edit movies_comedy function
- edit movie_size function
- create pretty_print function

goblynwtf commented on ruby.rb:
```diff
@@ -38,32 +38,29 @@ def rating_to_stars(rating)
-        .sort.reverse
-        .select { |m| m[:name].length > rightsize[5] }
+  movies.map { |m| m[:name] }
+        .sort_by(&:length).reverse
```
.reverse под сорт опусти

goblynwtf commented on ruby.rb:
```diff
@@ -38,32 +38,29 @@ def rating_to_stars(rating)
+end
+
+def pretty_print(m)
+  "Title:#{m[:name]}, (#{m[:realise]},#{m[:geners]} - #{m[:runtime]} "
```
лишний пробел в конце

goblynwtf reviewed (changes requested):
В основной твоей функции приведи ее в общему API

LastVegass committed:
fix issues

-fix function parse_txt_file
